src/get-markdown-notes.js

```diff
@@ -3,10 +3,22 @@ const path = require("path");
 
 module.exports = (pluginOptions) => {
   let notesDirectory = pluginOptions.notesDirectory || "content/brain/";
+  let notesFileExtensions = pluginOptions.notesFileExtensions;
```

Comment:
I think you could change this to `pluginOptions.notesFileExtensions || [".md", ".mdx"];`

Then you can delete 8-10, and the if check around 17-19. Plus, since this will have a default, it would be good for consistent behavior to always apply the filter rather than only applying it if manually set.

Reply:
I had`pluginOptions.notesFileExtensions || [".md", ".mdx"];` originally but this forced the user to provide all of the file extensions they wanted to use rather than being able to set `notesFileExtensions` to `null` in the config. I suppose it's unlikely that anything other than `md` and `mdx` files are going to be used so maybe it doesn't make sense to allow `null` and force the user to supply the additional extensions.

Reply:
I think that should be fine. I would rather it be explicit with it if the behavior is there. May add in a wildcard operator support later, but I doubt that should be needed.